src/components/PuzzleModal/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react"
+import "./index.scss"
+import { ReactComponent as ExitIcon } from "../../images/icons/circle-xmark-solid.svg"
+
+interface PuzzleProps {
+  children: React.ReactNode
+  question: string
+  answer: string
+}
+
+const link =
+  "https://docs.google.com/forms/d/e/1FAIpQLSfyGGDipFozDbQrmDBIoGvCwLcn5Lec3GMNnCgsRWKp9Q4syA/viewform"
+
+const PuzzleModal: React.FC<PuzzleProps> = ({
+  children,
+  question,
+  answer,
+}: PuzzleProps) => {
+  const [response, setResponse] = useState("")
+  const [success, setSuccess] = useState(false)
+  const [errorMessage, setErrorMessage] = useState("")
+  const [isPuzzleHidden, setIsPuzzleHidden] = useState(true)
+  const handleExit = () => setIsPuzzleHidden(true)
+  const hiddenOrNot = isPuzzleHidden
+    ? "puzzlemodal__container hidden"
+    : "puzzlemodal__container displayed"
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setResponse(event.target.value)
+  }
+
+  const handleSubmit = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === "Enter") {
+      if (response.toLowerCase() === answer) {
+        setSuccess(true)
+        setErrorMessage("")
+      } else {
+        setErrorMessage("Sorry, that's incorrect. Guess again")
+      }
+    }
+  }
+  const SuccessMessage = (
+    <div className='successMessage'>
+      That&apos;s correct!
+      <a href={link} target='_blank' rel='noreferrer'>
+        {" Click here "}
+      </a>
```

Comment:
So we have a space before and after in the `Click Here` let's remove those and add a space after "That's correct!" and "to earn your CruzCoins points!". Main reason is that the styles for the href are still being applied to those spaces.